File: btmanager/src/main/java/karrel/com/btconnector/btmanager/BluetoothListener.java
package karrel.com.btconnector.btmanager;

import android.bluetooth.BluetoothDevice;

import karrel.com.btconnector.chatmanager.BluetoothChatListener;

/**
 * Created by Rell on 2018. 1. 29..
 */

public interface BluetoothListener extends BluetoothChatListener {
    // 권한이 거부됨
    void deniedPermission();

    // 블루투스 사용가능 요청
    void requireEnableBt();

    // 기기 찾음
    void onSearchedDevice(BluetoothDevice device);
}

File: btmanager/src/main/java/karrel/com/btconnector/btmanager/BluetoothListener.kt
package karrel.com.btconnector.btmanager

import android.bluetooth.BluetoothDevice

import karrel.com.btconnector.chatmanager.BluetoothChatListener

/**
 * Created by Rell on 2018. 1. 29..
 */

interface BluetoothListener : BluetoothChatListener {
    // 권한이 거부됨
    fun deniedPermission()

    // 블루투스 사용가능 요청
    fun requireEnableBt()

    // 기기 찾음
    fun onSearchedDevice(device: BluetoothDevice?)
}
